e2e/cypress/integration/emoji/react_to_last_message_shortcut_spec.js

```diff
@@ -0,0 +1,575 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+import * as TIMEOUTS from '../../fixtures/timeouts';
+import * as MESSAGES from '../../fixtures/messages';
+import users from '../../fixtures/users.json';
+
+const newChannelName = `channel-react-to-last-message-${Date.now()}`;
+let channelId = '';
+let newChannel = {};
+
+describe('React to last message shortcut', () => {
+    before(() => {
+        cy.apiLogin('sysadmin');
+        cy.visit('/');
+
+        // # Get the current channels Id for later use such as posting message with other user
+        cy.getCurrentChannelId().then((id) => {
+            channelId = id;
+        });
```

Comment:
This might not be needed.

Reply:
This i am referencing in 
```
// # Post another message
cy.postMessageAs({sender: user1, message: 'Just another message', channelId});
```
to send message from another user